modules/rust_examples/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[workspace]
+resolver = "2"
+members = [
+   "RsExample",
+   "RsExampleUser",
+]
+
+[workspace.dependencies]
+# This is not used when building with cmake, but you can change this for local
+# development to point to a directory that works for you to build with cargo.
+everestrs = { path = "../everest-framework/everestrs/everestrs" }
```

Comment:
so this file would be managed by hand for the *cargo only* experience?

Reply:
would it be an option, if we supply a sample file here (like Cargo.toml.sample), which then can be copied and modified by the rust developer to Cargo.toml, and putting it into .gitignore, so it won't be uploaded?